Scripts/DCS-BIOS/lib/modules/aircraft_modules/MiG-29A.lua

```diff
@@ -366,7 +366,20 @@ MiG_29A:definePotentiometer("MASTER_CAUTION_BRIGHTNESS_KNOB", devices.INTLIGHTS_
 
 -- HSI
 
--- Combined indicator (VVI/Turn/Slip indicator)
```

Comment:
Is this in the right section? The combined VVI/Turn/Slip indicator is just to the right of the ADI on the panel